.gitignore

```diff
@@ -146,4 +146,7 @@ policy/
 
 # BATS
 _test/test_helper/
-_test/prometheus/rules.yaml
\ No newline at end of file
+_test/prometheus/rules.yamlcharts/pelorus/Chart.lock
```

Comment:
this line doesn't look right.

Reply:
Ah! I added the first path with VS code, which doesn't add line endings for the last line. But then the second path was `echo path >> .gitignore`, which meant it was on the same line. Treacherous. Will fix.